### DISCUSSION TITLE
Bun package manager with npmrc

### DISCUSSION THREAD

**Discussion by harm-matthias-harms** [renovatebot/renovate#30813] [Suggest an Idea]
### Tell us more.

Bun supports [.npmrc](https://bun.sh/docs/install/npmrc), well known from renovate's config files and npm package manager. It's not Bun's recommended way, but it would help to update Bun's lock file on a PR. We use private registries and need to manually update the lock file on every PR opened.

**Comment by rarkins** [comment]
Converted to feature request issue: https://github.com/renovatebot/renovate/issues/30926
